### PR TITLE
Add examples of GraphQL multiple mutations with single queries

### PR DESCRIPTION
**Pull request by tomowatt**
Currently the docs show how to use the GraphQL API to run single actions e.g., delete 1 Pipeline; but it is possible to write a single query that runs multiple mutations e.g., to deletes multiple Pipelines

Additionally, added examples of Archiving and Unarching Pipelines with a bit of refactoring to reduce repeating instructions to get the Pipeline ID as it's required for most of the examples

### CODE REVIEW

File: pages/apis/graphql/cookbooks/pipelines.md [hunk 1]
```diff
@@ -188,3 +214,70 @@ mutation UpdateSchedule {
...
+  })
+  {
+    pipeline {
+      id
+      name
+    }
+  }
+
+  pipeline2: pipelineArchive(input: {
+  id: "pipeline2-id"
```
Comment (gilesgas) [inline pages/apis/graphql/cookbooks/pipelines.md:255]: Indenting the `id:` bit a bit more ...

```suggestion
    id: "pipeline2-id"
```

**Comment by gilesgas** [comment]
Thanks very much for your docs contribution @tomowatt !

Just made one minor suggestion to merge in (above). Also, is there a process to unarchive multiple pipelines?

**Comment by tomowatt** [comment]
> Thanks very much for your docs contribution @tomowatt !
> 
> Just made one minor suggestion to merge in (above). Also, is there a process to unarchive multiple pipelines?

@gilesgas I can add one if required but it follows the same pattern as Archiving Multiple Pipelines, above, so was not sure if it would be too redundant given the minor changes i.e., `pipelineArchive` -> `pipelineUnarchive`
But if you feel there is merit for being added I'll get it added

**Comment by gilesgas** [comment]
> @gilesgas I can add one if required but it follows the same pattern as Archiving Multiple Pipelines, above, so was not sure if it would be too redundant given the minor changes i.e., pipelineArchive -> pipelineUnarchive
But if you feel there is merit for being added I'll get it added

I can see your point here, but for documentation, this should still be spelled out as a possibility (with at least a bare minimum explanation on how it's done) rather than expecting the reader to infer this knowledge.

I've compromised and added a brief section explaining how this is done from what you've described above. Hope this is OK?

**Review by gilesgas — APPROVED** [review]

**Comment by tomowatt** [comment]
> > @gilesgas I can add one if required but it follows the same pattern as Archiving Multiple Pipelines, above, so was not sure if it would be too redundant given the minor changes i.e., pipelineArchive -> pipelineUnarchive
> > But if you feel there is merit for being added I'll get it added
> 
> I can see your point here, but for documentation, this should still be spelled out as a possibility (with at least a bare minimum explanation on how it's done) rather than expecting the reader to infer this knowledge.
> 
> I've compromised and added a brief section explaining how this is done from what you've described above. Hope this is OK?

Thanks for that, will keep it in mind for the future!

**Pull request merged**